core/config/toml/types.go

```diff
@@ -1433,6 +1433,28 @@ func (m *MercurySecrets) ValidateConfig() (err error) {
 	return err
 }
 
+type EngineExecutionRateLimit struct {
+	GlobalRPS      *float64
+	GlobalBurst    *int
+	PerSenderRPS   *float64
+	PerSenderBurst *int
+}
+
+func (eerl *EngineExecutionRateLimit) setFrom(f *EngineExecutionRateLimit) {
```

Comment:
Nit: it's fine to use `e` here since it's the self variable; that's considered idiomatic go